### ISSUE TITLE
Location of ini files

### ISSUE THREAD

**Issue by bargisou** [ManiAm/VENTOS_Public#31]
What should be the location of omnetpp.ini and omnetpp_general.ini ,if i want to excecute the two_subsequent streets inside VENTOS_Public->example->getting started-> 1_two subsequent streets,the writing your first algorithm.
Right now the location of "omnetpp.ini" file is in the "getting started" folder and "omnetpp_general.ini" is inside "examples" folder.

**Comment by ManiAm** [comment]
yes, their locations is correct. Make sure that you are using the latest VENTOS manual! The include omnetpp_general at the top of the configuration does not have ../ any more in the Manual.

**Comment by bargisou** [comment]
Bu the error saying "Cannot open ini file 'omnetpp_general.ini" still persists even after removing ../from the configuration file

**Comment by ManiAm** [comment]
check your run configuration. If the problem persists, send me a screen shot of the run configuration that you are using to run the scenarios + copy of your omnetpp.ini file + error you get.

**Comment by bargisou** [comment]
If I remove the ../ from the ini file i am not able to acess the cofiguration name from the run configuration.
![img1](https://user-images.githubusercontent.com/35774990/39507781-3af53c68-4dfd-11e8-9bdb-0d5872d4ed7e.png)
![img2](https://user-images.githubusercontent.com/35774990/39507782-3b4177b8-4dfd-11e8-847d-f9f5a9f50638.png)
![img3](https://user-images.githubusercontent.com/35774990/39507783-3b72b152-4dfd-11e8-8b5b-9fe30a6f81fe.png)
![img4](https://user-images.githubusercontent.com/35774990/39507784-3bad2562-4dfd-11e8-8523-c63adb72a4a8.png)
If I keep the ../in the ini file then i am able to access the configuration name
![img5](https://user-images.githubusercontent.com/35774990/39507832-6eae67aa-4dfd-11e8-9019-cf0fa8f13be9.png)
![img6](https://user-images.githubusercontent.com/35774990/39507834-6edec3a0-4dfd-11e8-81c3-64f34d0512f2.png)
![img7](https://user-images.githubusercontent.com/35774990/39507835-6f0f15aa-4dfd-11e8-8339-274331cf818c.png)

**Comment by bargisou** [comment]
But when i try to run 7_platooning, there are no errors generated for that and the omnet++/qtenv,sumo  window is updated
successfully.But no vehicles movement could be observed
